src/Rings/mpoly-localizations.jl

```diff
@@ -92,9 +92,28 @@ Base.iterate(U::MPolyPowersOfElement, i::Int) = (i < length(U.a) ? (U.a[i+1], i+
 is_trivial(U::MPolyPowersOfElement) = (U == units_of(ambient_ring(U)))
 
 ### printing
+function Base.show(io::IO, ::MIME"text/plain", S::MPolyPowersOfElement)
+  io = pretty(io)
+  println(io, "Multiplicative subset")
+  print(io, Indent())
+  println(io, "of ", Lowercase(), ambient_ring(S))
+  print(io, "given by the powers of ")
+  print(io, "[")
+  join(io, denominators(S), ", ")
+  print(io, "]")
+  print(io, Dedent())
+end
+
 function Base.show(io::IO, S::MPolyPowersOfElement)
-  print(io, "powers of ")
-  print(io, denominators(S))
+  io = pretty(io)
+  if get(io, :supercompact, false)
```

Comment:
In supercompact, either print nothing at all or something stating that we are dealing with powers/products of elements.
The 'multiplicative set' should be left for multiple localizations, where we see more general multiplicative sets (unless we simplify them, which is not even on our agenda...)